Stripe/STPPaymentMethodTableViewCell.m

```diff
@@ -76,10 +86,10 @@ - (NSAttributedString *)buildAttributedStringForPaymentMethod:(id<STPPaymentMeth
 
 }
 
-- (NSAttributedString *)buildAttributedStringForCardBrand:(NSString *)brandString
-                                                    last4:(NSString *)last4
-                                                 selected:(BOOL)selected {
-    NSString *template = STPLocalizedString(@"%@ Ending In %@", @"{card brand} ending in {last4}");
+- (NSAttributedString *)buildAttributedStringWithName:(NSString *)brandString
+                                                last4:(NSString *)last4
+                                             selected:(BOOL)selected {
+    NSString *template = STPLocalizedString(@"%@ Ending In %@", @"{account name} ending in {last4}");
```

Comment:
We should update the logic in STPSource/STPCard's STPPaymentMethod `paymentMethodLabel` method to return the same thing here probably.